www/pages/stop/stop.html

```diff
@@ -1,44 +1,56 @@
 <ion-view>
-  <ion-nav-title>Departures: {{stop.Name}} ({{stop.StopId}})</ion-nav-title>
+  <ion-nav-title>{{stop.Name}} ({{stop.StopId}})</ion-nav-title>
   <ion-nav-buttons side="secondary">
-    <button id="like"  ng-class="{'button button-icon icon ion-ios-heart': liked, 'button button-icon icon ion-ios-heart-outline': !liked}" ng-click="liked= !liked; toggleHeart(liked)"></button>
+    <button id="like" ng-class="{'button button-icon icon ion-ios-heart': liked, 'button button-icon icon ion-ios-heart-outline': !liked}"
+                      ng-click="liked= !liked; toggleHeart(liked)">
+    </button>
   </ion-nav-buttons>
-  <ion-content direction="y" scrollbar-y="false">
+  <ion-content>
     <ion-refresher pulling-text="Pull to refresh..." on-refresh="refresh()"></ion-refresher>
     <!-- For every route, group it and allow it to be toggled open or closed -->
-    <div class="item item-divider item-positive">
-      <center>
-        Tap a route to view upcoming departures
-      </center>
+    <div class="item item-divider item-positive center">
+      Tap a route to view more departures
     </div>
     <ion-list ng-repeat="routeDepartures in departuresByRoute">
-      <ion-item ng-click="toggleGroup({{routeDepartures.RouteId}})" ng-class="{active: isGroupShown({{routeDepartures.RouteId}})}">
-        <div style="font-weight: bold;font-size: 175%; color: #{{routeList[routeDepartures.RouteId].Color}};">
-          <center>
+      <ion-item ng-click="toggleGroup({{routeDepartures.RouteId}}, {{routeDepartures.Departures.length}})"
+                      ng-class="{active: isGroupShown({{routeDepartures.RouteId}})}"
+                      style="background-color: #{{routeList[routeDepartures.RouteId].Color}};">
+        <div class="row item-text-wrap white">
+          <div class="col route-shortname">
             {{routeList[routeDepartures.RouteId].RouteAbbreviation}}
-                <i class="icon" style="padding-left: 3%;" ng-class="isGroupShown({{routeDepartures.RouteId}}) ? 'ion-ios-arrow-down' : 'ion-ios-arrow-up'" style="margin-right: 10px;"></i>
-          </center>
+          </div>
+          <div class="col-50">
+            {{routeDepartures.Departures[0].Trip.InternetServiceDesc}}
+          </div>
+          <div class="col-33 text-right">
+            <span>
+              {{routeDepartures.Departures[0].Times.eExact}}
+            </span>
+            <p style="color: white">
+              ({{routeDepartures.Departures[0].Times.eRelativeNoPrefix}})
+            </p>
+          </div>
         </div>
       </ion-item>
+      <div class="item item-divider no-more-departures"
+                      style="background-color: #{{routeList[routeDepartures.RouteId].Color}};"
+                      ng-if="onlyOneDeparture.indexOf(routeDepartures.RouteId) !== -1">
+        <span>No other upcoming departures</span>
```

Comment:
This should probably go in the expanded div, it shows when the stop is expanded but will not disappear when the stop is toggled again.